ISSUE_TEMPLATE.md

```diff
@@ -1,22 +0,0 @@
-<!--
-Thank you for reporting an issue.
-
-This issue tracker is for bugs and issues found within Mono's core.
-If you require more general support please file an issue on our help
-repo. https://github.com/mono/help
```

Comment:
should we keep this?

Reply:
I don't know what's the purpose of that, seems like what should be on mono github wiki